RecoLocalTracker/SiStripClusterizer/plugins/alpaka/SiStripRawToClusterAlgo.dev.cc

```diff
@@ -0,0 +1,1096 @@
+#include <alpaka/alpaka.hpp>
+
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/warpsize.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/workdivision.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/prefixScan.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/moveToDeviceAsync.h"
+
+#include "RecoLocalTracker/SiStripClusterizer/interface/ClusterChargeCut.h"
+#include "RecoLocalTracker/SiStripClusterizer/interface/SiStripClusterizerConditionsStruct.h"
+
+#include "EventFilter/SiStripRawToDigi/interface/SiStripFEDBuffer.h"
+#include "EventFilter/SiStripRawToDigi/interface/SiStripFEDBufferComponents.h"
+
+#include "SiStripRawToClusterAlgo.h"
+
+// Generic raw unpackers
```

Comment:
I'm curious - why all the offsets are used XOR 7 ?

Reply:
OK 🤔 

Instead of duplicating those functions here, could you mark them as `constexpr` in `SiStripFEDBuffer.h` and reuse them from there ?

Reply:
And add a reference for those that you have to duplicate.